### PR TITLE
Add slack channel notifications

### PR DESCRIPTION
**Pull request by jkrauska**
Often you need to track a different slack channel for Team Chat vs Team Notifications -- this change adds a column to track that and adds api and UI support.

### CODE REVIEW

**Comment by jkrauska** [comment]
Wasn't sure on best approach to /update/ schema cleanly.. Please advise.

File: e2e/test_teams.py [hunk 1]
```diff
@@ -101,6 +103,7 @@ def test_api_v0_update_team(team):
     re = requests.put(api_v0('teams/'+team_name), json={'name': new_team_name,
                                                         'email': email,
                                                         'slack_channel': slack,
+                                                        'slack_channel_notificationse': slack_notifications,
```
Comment (minhaz1) [inline e2e/test_teams.py:106]: Should this say `slack_channel_notifications` instead of `slack_channel_notificationse`?

Comment (jkrauska) [reply e2e/test_teams.py]: thankyou

**Comment by jkrauska** [comment]
> Wasn't sure on best approach to /update/ schema cleanly.. Please advise.

It seems that CI doesn't support schema updates per-se, so I pushed my schema changes in to the original sql.

**Pull request merged**